modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -340,3 +337,35 @@ Include Password Field:: Shows the password field.
 Extra Field Set:: Specifies extra fields.
 Registrations Require Approval:: Select whether registration requires approval.
 Approver ID:: Specifies the approver ID.
+
+== Filter Conditions [[filter-conditions]]
+
+The API Carousel component and the API Catalog component use the parameter *Filter Condition (Advanced)* to show only the APIs that meet the condition. A condition is a standard search in Salesforce Object Query Language (SOQL).
+
+You can search these fields:
```

Comment:
Search these fields for what? the condition? Do we need to specify?

Reply:
The search can test those fields to see if they match the condition. Updated this to clarify.